Eksempel app med all .less automatisk importert.

hagenek opened the pull request:
- **did npm init, setting up vite**
- **made readme for exampleapp**
- **Rudimentary example app made, with script that imports all less files**

hagenek committed:
feat: Rudimentary example app made, with script that imports all less files
made readme for exampleapp
did npm init, setting up vite

hagenek committed:
Optimization attempt

hagenek committed:
Add oxlint, add text about usage

hagenek committed:
Sidebar scaffolding done

hagenek committed:
Add sidebar, create alpha version of link component

dagfrode commented on examples/sbanky/README.md:
```diff
@@ -0,0 +1,205 @@
+      less: {
+        javascriptEnabled: true,
+        math: 'always',
+        additionalData: '@import "@sb1/ffe-core/less/colors.less";'
```
hvorfor `colors.less` og ikke `ffe.less`?

hagenek replied:
endret til ffe.less, det er bedre ja.

dagfrode commented on structure.txt:
```diff
@@ -0,0 +1,603 @@
+├── CODE_OF_CONDUCT.md
+├── CONTRIBUTING.md
+├── LICENSE-fonts.md
+├── LICENSE.md
```
Gir det mening å ha med denne filen?

hagenek replied:
Jeg bruker den som kontekst til LLM-en, men kan legge den i .gitignore. Det er jo stale data.

dagfrode reviewed (commented):
Synes det ser bra ut. Føler meg ikke helt klar på å bedømme om det er der at det kans merges inn enda eller ikke.

Kan være det bør være en annen bakgrunnsfarge, blir noen føringer med semantiske farger nå snart.

Når vi lager eksempler med forms, prøve å lage eksempler med Zod og reack-form-hook?

hagenek committed:
fix(ffe-vite-less): fikser tailwind og FFE LESS-import i eksempelappen

- Legger til riktig konfigurasjon for tailwind
- Fjerner duplisert LESS-konfigurasjon
- Fikser typefeil i vite-config
- Legger til manglende LESS-stier for FFE-komponenter
